### PR TITLE
docs: Update Polish readme

### PR DESCRIPTION
**Pull request by mbiesiad**
# Description

Updated Polish readme - current version 1.1.1.

## Checklist

- [x] The title of my PR starts with a [Conventional Commit] prefix (`fix:`, `feat:`, `docs:` etc).
- [x] I have read the [Contributor Guide] and followed the process outlined for submitting PRs.
- [ ] I have updated/added tests for ALL new/updated/fixed functionality.
- [ ] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [ ] I have updated/added relevant examples in `examples`.

## Breaking Change

- [ ] Yes, this is a breaking change.
- [x] No, this is *not* a breaking change.

## Related Issues
https://github.com/flame-engine/flame/issues/1195#issuecomment-1106344807

Best wishes,

### CODE REVIEW

**Comment by spydon** [comment]
@kjagiello could you review this? :slightly_smiling_face:

File: i18n/README-PL.md [hunk 1]
```diff
@@ -1,107 +1,162 @@
...
+Oto niektóre z kluczowych dostępnych funkcji:
 
-Mamy też najczęściej zadawane pytania - [FAQ](FAQ.md), więc najpierw wyszukaj tam swoje pytanie.
+ - Pętla gry.
+ - System komponent/obiekt (FCS).
+ - Efekty i cząsteczki.
+ - Wykrywanie kolizji.
+ - Obsługa gestów i wprowadzania danych
+ - Obrazy, duszki (sprites) i arkusze duszka
+ - Ogólne narzędzia ułatwiające programowanie.
```
Comment (kjagiello) [inline i18n/README-PL.md:61]: Brace yourselves. Huge nitpick incoming 😅

Every of these points should probably end with a period, the same way they do in the English docs:

> - A game loop.
> - A component/object system (FCS).
> - Effects and particles.
> - Collision detection.
> - Gesture and input handling.
> - Images, animations, sprites, and sprite sheets.
> - General utilities to make development easier.

```suggestion
 - Pętla gry.
 - System komponent/obiekt (FCS).
 - Efekty i cząsteczki.
 - Wykrywanie kolizji.
 - Obsługa gestów i wprowadzania danych.
 - Obrazy, duszki (sprites) i arkusze duszka.
 - Ogólne narzędzia ułatwiające programowanie.
```

**Review by kjagiello — COMMENTED** [review]
Looking good! Just a tiny fix and this is good to go 🚀

File: i18n/README-PL.md (hunk 1 above)
Comment (mbiesiad) [reply i18n/README-PL.md]: Good point, thanks a lot! ✨

**Review by spydon — APPROVED** [review]

**Pull request merged**